services/frontend-service/src/ui/components/ServiceLane/ServiceLane.scss

```diff
@@ -1,34 +1,40 @@
-.service-lane__header{
-  background: var(--mdc-theme-primary);
-  color: var(--mdc-theme-on-primary);
-  display: flex;
-  justify-content: space-between;
-  padding: 13px 20px;
-  border-radius: 10px;
-  height: 62px;
+.service-lane__header {
+	background: var(--mdc-theme-primary);
+	color: var(--mdc-theme-on-primary);
+	display: flex;
+	justify-content: space-between;
+	padding: 13px 20px;
+	border-radius: 10px;
+	height: 62px;
 
-  .service__name {
-    font-weight: 700;
-    font-size: 28px;
-    line-height: 48px;
-    display: flex;
-    align-items: center;
-  }
-  .service-action {
-    border: 1px solid var(--mdc-theme-on-primary);
-    border-radius: 10px;
-    margin-left: 51px;
-    color: var(--mdc-theme-on-primary);
-    .mdc-button__label{
-      padding-left: 1em;
-      font-weight: 700;
-      font-size: 14px;
-      line-height: 24px;
-    }
-  }
+	.service__name {
+		font-weight: 700;
```

Comment:
Again fix the spacing here

Reply:
@leandro-freiheit please create a story to have a linter on scss in kuberpult.

Reply:
https://github.com/freiheit-com/kuberpult/issues/382

Reply:
done!